code/modules/research/designs/mecha_equipment.dm

```diff
@@ -110,6 +110,10 @@
 	name = "CH-LC \"Solaris\" laser cannon"
 	build_path = /obj/item/mecha_parts/mecha_equipment/weapon/energy/laser/heavy
 
+/datum/design/research/item/mecha/weapon/auto_laser
```

Comment:
No

Reply:
I mean, what is the point of adding a weapon if it is not obtainable?